### ISSUE TITLE
[CoE Starter Kit - BUG] [Child] Admin | Sync Logs - Action 'Get_a_row_by_ID' failed

### ISSUE THREAD

**Issue by JanetK12** [microsoft/coe-starter-kit#6036]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

We noticed that the flow [Child] Admin | Sync Logs Action often fails. From the "Run Details" it is stated that the problem is with the 'Get_a_row_by_ID' action. 
![image](https://github.com/microsoft/coe-starter-kit/assets/113427788/068825e1-8efc-41ff-8b00-fcc98a410124)
<img width="773" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/113427788/e329870d-94dd-40db-97ef-e7674adbbf89">
However, we also noticed that the failure happens every time when the action "If NextPageURI not blank, need smaller chunks" is "true", so it takes the left brunch where the action 'Get_a_row_by_ID' is located and fails.
![image](https://github.com/microsoft/coe-starter-kit/assets/113427788/ec283468-4fea-4fd3-91ce-afa13c4dc9e2)

If the action "If NextPageURI not blank, need smaller chunks" is "false", it goes to the right brunch and the run is successful.
![image](https://github.com/microsoft/coe-starter-kit/assets/113427788/246f7d98-c455-430e-b614-c12544008a8c)
 
So we assume that the problem can be with the left branch of the actions - "Get Row - Send notice of required chunking".
![image](https://github.com/microsoft/coe-starter-kit/assets/113427788/37a276ee-2e03-4299-86e4-460b038b89fe)

### Expected Behavior

To solve the issue, so that we will not have the errors with the [Child] Admin | Sync Logs, as we have them almost every day. 

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

3.3

### What app or flow are you having the issue with?

[Child] Admin | Sync Logs

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Hello. The custom connector methodology to collect audit logs has been deprecated in favor of an Http methodology that supports paging. As a result, improvements in this space will not be made and the custom connector will be removed in August. 

If you would like to keep the custom connector methodology as the way you gather audit logs, and make these changes locally, please see [How to maintain custom connector methodology to gather audit logs](https://github.com/microsoft/coe-starter-kit/issues/6017)

If you would like to convert to the new Http methodology please see [How to convert audit logs from custom connector to http format](https://github.com/microsoft/coe-starter-kit/issues/6009
)

**Comment by JanetK12** [comment]
Hi! Thank you for your answer!
Could I ask, am I understanding that right that we have some steps that we can do manually as you send the links not to wait for the upgrade by receiving these errors.
But also we can just leave as it is for now (not to get the unmanaged layer) and wait for the upgrade that will be in August that will automatically delete custom connector, so after the upgrade there will not be similar errors?

**Comment by Jenefer-Monroe** [comment]
No you do have manual steps to do one way or another. The audit log solution you have now will need deleted before the August upgrade will be able to occur as it has a reference to the custom connector. 

Are you actively using / gather audit logs today and hitting this bug occassionally? Or are you just getting started with audit logs?

**Comment by JanetK12** [comment]
Hi @Jenefer-Monroe !
We have been using them for a long time. The fails started to occur after the last upgrade we did - the April upgrade (version 3.3). Because after the April audit logs upgrade the custom connector needed to be updated - we did that. After that the runs became mostly successful, but sometimes they fails. 
The fails in the 14.07 were because of the upgrade. After that we updated the custom connector (Client ID, Client secret, etc ) and the funs became successful, however, as you can see sometimes it fails, even 08.07 all the runs failed with the errors that I provided at the beginning od this ticket. 
<img width="600" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/113427788/19ac67e4-8513-4dab-99bb-17fea7fd336e">

**Comment by Jenefer-Monroe** [comment]
Got it ok, I was just checking as that will help you decide your path. 

If you would like to keep using the CC, you will need to get a local copy of it and the rest of the solution, then you can fix it up as you desire as you suggest above. The steps to do this are here: [How to maintain custom connector methodology to gather audit logs](https://github.com/microsoft/coe-starter-kit/issues/6017)

If you would like to convert to the new Http methodology please see [How to convert audit logs from custom connector to http format](https://github.com/microsoft/coe-starter-kit/issues/6009). In this methodology there is no "chunking" required as the pagination is supported by the call itself.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by JanetK12** [comment]
Hi @Jenefer-Monroe !
Thank you for your previous answer! I apologize, I forgot to answer. 
So am I understanding that right that I need to create the copy of the whole "Center of Excellence - Audit Logs" solution and then recreate the steps as they are in the link you provided? I am just not sure till the end what means by the word "local copy"? Does that means that I need to install again the "Center of Excellence - Audit Logs" solution as a second solution? What about the first "Center of Excellence - Audit Logs" solution then? Do I need to delete it?
And also, if I am not mistaken, as in August the CC will be removed, that means if we decide to you the CC version, then we will not be able to upgrade this solution anymore, right?
Thank you in advance!

**Comment by Jenefer-Monroe** [comment]
No problem. I've tried to answer your question here but not sure if I did. 

You do not need to create a copy yourself no. The steps outlined in https://github.com/microsoft/coe-starter-kit/issues/6017 will show you how to import a new copy of the custom connector and flows etc so that you are not tied to the old system, as well as when to delte the old audit logs (if you still had them installed)
You will not be blocked from installing August as you will not be tied to the old custom connector.
